dataikuapi/dss/dataset.py

```diff
@@ -859,3 +870,78 @@ def already_exists(self):
             return True
         except Exception as e:
             return False
+
+
+class DSSDatasetInfo(object):
+    """
+    Info class for a DSS dataset (Read-Only).
+    Do not instantiate this class directly, use :meth:`DSSDataset.get_info`
+    """
+
+    def __init__(self, dataset, info):
+        """
+        :type dataset: :class:`DSSDataset`
+        :type info: dict
+        """
+        self.dataset = dataset
+        self.info = info
+
+    def get_raw(self):
+        """
+        Get the raw dataset full information as a dict
+
+        :return: the raw dataset full information
+        :rtype: dict
+        """
+        return self.info
+
+    def get_raw_last_build(self):
+        """
+        Get the last build information of the dataset, as a raw dict, or None if there is no last build information.
+
+        :return: the last build information
+        :rtype: dict or None
+        """
+        return self.info.get("lastBuild", None)
+
+    def get_last_build_start_time(self, as_date=False):
```

Comment:
I would put as_date = True by default.

Reply:
Actually, to be consistent with scenario.py, I would:

* Remove the option as_date.
* Always return a date
* Make it a property

check scenario.py 510 and 517

Reply:
done 👍🏻